src/tests/reducers/userReducer.test.js

```diff
@@ -47,6 +50,12 @@ describe('userReducer', () => {
     expect(currentState).toEqual({
       ...initialState,
       registerUserSuccess: true,
+      registerUserError: {},
```

Comment:
The `...initialState` caters for the other key-value pairs so you do not need to define them unless you're changing them intentionally.